pkg/cloud/azure/actuators/machineset/azure_instance_types.go

```diff
@@ -18,2268 +18,2862 @@ limitations under the License.
 package machineset
 
 type instanceType struct {
-	InstanceType string
-	VCPU         int64
-	MemoryMb     int64
-	GPU          int64
+	InstanceType          string
+	VCPU                  int64
+	MemoryMb              int64
+	GPU                   int64
+	AcceleratedNetworking bool
```

Comment:
How was this list populated, please don't say by hand 😅 

This list was copied from the Cluster Autoscaler repo originally, if we need to update it, we will want to just copy paste it again. Can this be regenerated later?

Reply:
I attempted to work in the code from upstream that checks the capabilities based upon instance type and region. I was unsuccessful getting the libraries to play nicely.

I proceeded to copy the azure_instance_types file from the upstream location and then added the upstream provider code to check whether the VM supports accelerated networking. Code can be found here -> https://github.com/kwoodson/azurevmtypes. The only downside to this approach is that the check uses a region. There might be some discrepancies in the region that I used (eastus) to check whether the VMs support this feature as regions differ at times with their supported instance types. I think this approach is safe until we move to the upstream provider.

Reply:
> I was unsuccessful getting the libraries to play nicely.

Any recollection at this point on why they didn't play nicely? Or maybe a link to the upstream code you were trying to copy?

Looking at this generator code, we should probably be including this within our repo and running it periodically 🤔